live-build/misc/migration-scripts/dx_upg_stress_options

```diff
@@ -32,9 +32,14 @@ __STRESS_OPTIONS_JSON=$(
 		    "err_msg": "Stress option triggered to fail hotfix check.",
 		    "auto_unset": true
 		},
-		"STRESS_DX_VERIFY_FAIL_AFTER_TEST_MIGRATION": {
+		"STRESS_DX_VERIFY_FAIL_AFTER_UPGRADE_VERIFY": {
```

Comment:
I talked to Palash and it does seems like it will overlap with the upgrade-specific testing. I think I'm going to revert to the old name to minimize the amount of qa-gate changes we'll need.